### ISSUE TITLE
Using mobilenet as face detection model

### ISSUE THREAD

**Issue by SirVizzy** [vladmandic/human#473]
Hi. 

At the moment I am using FaceAPI with the MobileNetV1 model for face detection in images. I have off-loaded the work to a WebWorker which works fine when WebGL is supported. However, on my phone the tensorflow backend keeps being set to `cpu`. Which is problematic as it's stupendously slow; I would need some fallback mechanism where `opengl => wasm => cpu`.

Since FaceAPI wasn't originally designed to be used with a worker, I would like to try Human.

I have tried numerous models to see which fits my use-case best. And I seem to be getting the best results using the MobileNetV1 model (better than Blazeface). So my question is, is it possible to use the MobileNetV1 model instead? Also does this package come with proper tree shaking?

**Comment by SirVizzy** [comment]
I did find the following in the documentation; https://github.com/vladmandic/human-models. However, I could not find the model I was looking for in there. I am looking for a model that works for both short- and longer range detection which high accuracy and low error rates.

**Comment by vladmandic** [comment]
did you evaluate face detection model in human as-is? i don't have plan to implement much older mobilenetv1 in human.

**Comment by SirVizzy** [comment]
Yes. Blazeface isn't cutting it with images with groups of people (for either sparse or regular). I do see a `mobilefacenet` in https://github.com/vladmandic/human-models/tree/main/models which has equal amount of bytes to that model. Is it similar? Do you know the origin?

**Comment by vladmandic** [comment]
> Most problematic issue with blazeface that I am encountering is a ton of false detections

did you try changing params? default min-confidence is 0.2 which is really low as assumption is that detected faces would be further filtered out by analysis part. if you're running just detect part, then you need to tweak it to your liking.

if you want to use alternative face detection, see `demo/faceid` for config syntax. although at this time only blazeface is fully supported.
